cmd/xpumanager_sidecar/README.md

```diff
@@ -8,6 +8,7 @@ Table of Contents
   * [Install XPU-Manager with the Sidecar](#install-xpu-manager-with-the-sidecar)
   * [Install Sidecar to an Existing XPU-Manager](#install-sidecar-to-an-existing-xpu-manager)
 * [Verify Sidecar Functionality](#verify-sidecar-functionality)
+* [Use HTTPS with XPU-Manager]()
```

Comment:
XPU Manager project writes the name everywhere without '-': https://github.com/intel/xpumanager

so I think this doc should also write it like that.

And this entry should also have link to the corresponding section.